templates/tfengine/components/cicd/configs/tf-deletion-check.sh

```diff
@@ -13,34 +13,43 @@ set -e
 
 reject_plan_deletions="${1}"
```

Comment:
The result looks super nice in the cloud build run. Let's move it to the plan presubmit job as the last step. And we can remove the reject_plan_deletions attr and make it just part of that normal speculative plan run and give users a warnings for potential deletion.